{"content": {"suites": {"0": {"status": {"total_pass": 1, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 3, "total_error": 0}, "tests": {"0": {"status": "PASS", "message": "", "test_name": "test_15_SetupOs", "rerun": "0"}, "1": {"status": "FAIL", "message": "   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"KEA00\"]\"}\n     (Session info: chrome=112.0.5615.49)\n   Stacktrace:\n   0   chromedriver                        0x0000000102e9fae8 chromedriver + 4958952\n   1   chromedriver                        0x0000000102e977b3 chromedriver + 4925363\n   2   chromedriver                        0x0000000102a591d6 chromedriver + 475606\n   3   chromedriver                        0x0000000102a9ca20 chromedriver + 752160\n   4   chromedriver                        0x0000000102a9cc81 chromedriver + 752769\n   5   chromedriver                        0x0000000102ae0f64 chromedriver + 1032036\n   6   chromedriver                        0x0000000102ac2c3d chromedriver + 908349\n   7   chromedriver                        0x0000000102ade25b chromedriver + 1020507\n   8   chromedriver                        0x0000000102ac29e3 chromedriver + 907747\n   9   chromedriver                        0x0000000102a8caef chromedriver + 686831\n   10  chromedriver                        0x0000000102a8e00e chromedriver + 692238\n   11  chromedriver                        0x0000000102e5b9ad chromedriver + 4680109\n   12  chromedriver                        0x0000000102e60c12 chromedriver + 4701202\n   13  chromedriver                        0x0000000102e6779f chromedriver + 4728735\n   14  chromedriver                        0x0000000102e61b7a chromedriver + 4705146\n   15  chromedriver                        0x0000000102e3427e chromedriver + 4518526\n   16  chromedriver                        0x0000000102e81908 chromedriver + 4835592\n   17  chromedriver                        0x0000000102e81a87 chromedriver + 4835975\n   18  chromedriver                        0x0000000102e9100f chromedriver + 4898831\n   19  libsystem_pthread.dylib             0x00007ff815ce31d3 _pthread_start + 125\n   20  libsystem_pthread.dylib             0x00007ff815cdebd3 thread_start + 15\n       assert False\n", "test_name": "test_16_LoginSpv", "rerun": "0"}, "2": {"status": "FAIL", "message": "           ConnectionRefusedError: [Errno 61] Connection refused\n       urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x12242f160>: Failed to establish a new connection: [Errno 61] Connection refused\n       urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=60432): Max retries exceeded with url: /session/3833bc95eb0043523daae31f94639b84/timeouts (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x12242f160>: Failed to establish a new connection: [Errno 61] Connection refused'))\n", "test_name": "test_17_AksesMenuSpv", "rerun": "0"}, "3": {"status": "FAIL", "message": "           ConnectionRefusedError: [Errno 61] Connection refused\n       urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x126df0220>: Failed to establish a new connection: [Errno 61] Connection refused\n       urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=60432): Max retries exceeded with url: /session/3833bc95eb0043523daae31f94639b84/timeouts (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x126df0220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n", "test_name": "test_18_KirimLaporanSpv", "rerun": "0"}}, "suite_name": "4LaporanBulanan/7.Laporan7A/Fungsional/test_Laporan7A.py"}}}, "date": "April 12, 2023", "start_time": 1681285641.170321, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "1", "fail": "3", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "4"}